util/daemonize.c

```diff
@@ -62,6 +62,10 @@ daemonize(const char * spid)
 				/* EOF -- the child died without poking us. */
 				goto err1;
 			case 1:
+				/* We don't need to read from this any more. */
```

Comment:
I added this in #467, which you haven't commented on.

(I just wanted to make sure that you're fully aware of this change)